src/api/article/localized/controller.js

```diff
@@ -9,10 +9,17 @@ import LocalizedArticle from './model';
 export const create = async ({ params: { articleId }, body }, res, next) => {
```

Comment:
I found that this method is unused

Reply:
Do you want us to remove it? We can keep it just in case, it has test coverage and everything (I guess)

Reply:
We can add comment about it